src/Controller/ProjectsController.php

```diff
@@ -38,10 +40,17 @@ public function list(ProjectRepository $repository, Request $request): Response
             $repository->addConditionState($projectQuery, $state);
         }
 
+        if ($shuttle) {
+            $repository->addConditionShuttle($projectQuery, $shuttle);
+        }
+
         $projects = $projectQuery->getQuery()->execute();
 
+        $shuttles = $shuttleRepository->findAll();
+
         return $this->render('projects/list_project.html.twig', [
             'projects' => $projects,
+            'shuttles' => $shuttles,
```

Comment:
```suggestion
            'filters' => ['shuttles' => $shuttleRepository->findAll()],
```

add all the filters values only in a sub-element to ease comprehension of variables